docs/language/crypto.md

```diff
@@ -107,7 +107,10 @@ The API of the Crypto contract is:
 pub contract Crypto {
 
     // Hash the data using the given hashing algorithm and returns the hashed data.
-    pub fun hash(_ data: [UInt8], algorithm: HashAlgorithm): [UInt8]
+    pub fun hash(data: [UInt8], algorithm: HashAlgorithm): [UInt8]
```

Comment:
Any reason for adding the argument label? Usually the main argument that the function operates on doesn't need it.

Adding it would be a breaking change, so I'd recommend leaving it away / as-is, and also removing it from `hashWithTag`

Reply:
The new `hashWithTag` has more parameters. I was thinking that having parameters with labels as well as without labels, would be a bit confusing to the users. So this change was for consistency with the other method.

>  Usually the main argument that the function operates on doesn't need it.

But if that's a best practice we follow for Cadence, then yeah we can keep it as is.

Reply:
Reverte it in 0633d07